docs/searching-for-UiTPAS-events.md

```diff
@@ -0,0 +1,52 @@
+# Searching for UiTPAS events
+
+This mini-guide will explain how to search for UiTPAS events using the [Search API](https://documentatie.uitdatabank.be/content/search_api_3/latest/searching.html).
+
+> ##### Getting the organizerIds
+>
+> The Search API allows you to search in all UiTPAS events.
+> Mostly it only makes sense to search for events of the UiTPAS organizer the current user or client has access to.
+>
+> That's why you should first check which organizers you currently have UiTPAS permissions for using the [GET /permissions](https://docs.publiq.be/docs/uitpas/b3A6NDM0MjM5NzY-get-permissions) endpoint.
```

Comment:
You can also link to the docs for that endpoint with a relative link:
```
/reference/uitpas.json/paths/~1permissions/get
```

You can get this link by right-clicking the endpoint in the list of API endpoints in Stoplight Studio and clicking "Copy path" which will get you `reference/uitpas.json/paths/~1permissions`. You then need to suffix it with `/` and the HTTP method, so in this case `/get`.

It's a bit complex sadly, but it makes the links a bit more reliable in case the URL structure that Stoplight uses ever changes or we switch documentation solutions. (Although we need redirects then anyway)

This only works for links to API endpoints inside the same project though (so only UiTPAS API endpoints in this case).

Reply:
We do have dead link checkers on the Markdown files though, so if this is too complex I guess we can just use absolute URLs that you can copy-paste from the browser.

Only downside is that you can then only link to endpoints that are already merged into the Stable branch.